posthog/migrations/0526_add_features.py

```diff
@@ -0,0 +1,75 @@
+# Generated by Django 3.2.18 on 2023-06-22 11:08
+
+from django.db import migrations, models
+from django.utils import timezone
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0525_hog_function_transpiled"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="Feature",
+            fields=[
+                ("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                ("team", models.ForeignKey(on_delete=models.deletion.CASCADE, to="posthog.team")),
+                ("name", models.CharField(blank=True, max_length=400)),
```

Comment:
should this be a nullable (i.e. `Blank`) field?  Feels like we should always have this IMO.  The tests enforce it too, so why not just make it non-nullable.

Reply:
Nope, thanks for catching